docs/core/porting/winforms.md

```diff
@@ -87,7 +87,7 @@ Next, you need to create the **MyFormsCore.csproj** project in the **MyFormsAppC
 
 If you don't want to create the project file manually, you can use Visual Studio or the .NET Core SDK to generate the project. However, you must delete all other files generated by the project template except for the project file. To use the SDK, run the following command from the **SolutionFolder** directory:
 
-```cli
+```dotnetcli
```

Comment:
We don't have such an identifier yet. But I already suggested we have one added. Please apply console to all of these for now.

```suggestion
```console
```

Reply:
It is actually listed in the contribution guide, in the [how-to-write-use-markdown#code-snippets](https://docs.microsoft.com/en-us/contribute/how-to-write-use-markdown#code-snippets) page.

And it seems to work correctly online on this page: [deploy-with-cli](https://docs.microsoft.com/en-us/dotnet/core/deploying/deploy-with-cli#sample-app)

Reply:
Awesome! Glad to see it available 😄